ajcore.20160503.083804.142.txt

```diff
@@ -0,0 +1,337 @@
+---- AspectJ Properties ---
```

Comment:
The file is removed. Did i need to do anything else? It doesn't seem to come back upon rebuild.